### PR TITLE
chore(renovatebot): automerge @types/node

### PR DESCRIPTION
**Pull request by Xavientois**
**Fixes or implements VF-3530**

### Brief description. What is this change?

Enable automerge for `@types/node`

### Checklist

- [ ] this is a breaking change and should publish a new major version
- [ ] appropriate tests have been written

### CODE REVIEW

**Review by xavidop — CHANGES_REQUESTED** [review]
this should be in the default.json file, following this format: https://github.com/voiceflow/renovate-config/blob/aee538b5c07829f23935973f67ec38d6293b337d/default.json#L16

**Comment by Xavientois** [comment]
> this should be in the default.json file, following this format:
> 
> https://github.com/voiceflow/renovate-config/blob/aee538b5c07829f23935973f67ec38d6293b337d/default.json#L16

I'm not sure I understand. Are you saying that I should move the whole `@types/node` config to the `default.json` file or that I should enable automerge for everything?

**Comment by Xavientois** [comment]
> just move it to the `deafult.json` file

To avoid conflicts, I will wait until https://github.com/voiceflow/renovate-config/pull/8 is approved and merged before moving this config to the `default.json` file.

**Review by trs — APPROVED** [review]

File: default.json [hunk 1]
```diff
@@ -21,6 +21,17 @@
       "matchPackagePatterns": ["*"],
       "enabled": false
     },
+    {
+      "matchManagers": [
+        "npm"
+      ],
+      "matchPackageNames": [
+        "@types/node"
```
Comment (xavidop) [inline default.json:29]: only for minor patch and pin versions

**Review by xavidop — CHANGES_REQUESTED** [review]

File: default.json (hunk 1 above)
Comment (Xavientois) [reply default.json:29]: Oops, sorry. Forgot to add my changes to the commit

**Review by xavidop — APPROVED** [review]

**Pull request merged**